airflow/gcp/example_dags/example_gcs.py

```diff
@@ -127,7 +128,14 @@
         task_id="delete_files", bucket_name=BUCKET_1, objects=[BUCKET_FILE_LOCATION]
     )
 
+    delete_bucket = GCSDeleteBucketOperator(task_id="delete_bucket", bucket_name="tosia-lubi-kotki")
+
     [create_bucket1, create_bucket2] >> list_buckets >> list_buckets_result
     [create_bucket1, create_bucket2] >> upload_file
     upload_file >> [download_file, copy_file]
     upload_file >> gcs_bucket_create_acl_entry_task >> gcs_object_create_acl_entry_task >> delete_files
+    create_bucket1 >> delete_bucket
```

Comment:
I am afraid this can delete bucket before other tasks that may rely on it. Have you run this DAG?

Reply:
It was working on my machine but I will double check.